jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java

```diff
@@ -60,6 +62,7 @@
 	public void setup() {
 	    // disable auto init of timers since ejb timer service supports persistence of timers
 	    System.setProperty("org.jbpm.rm.init.timer", "false");
+	    logger.info("Using local cache for EJB timers: {}", (USE_LOCAL_CACHE ? "true" : "false"));
```

Comment:
Why do we need to use conditional expression and the print true or false, isn’t it enough to directly priny USELOCALCACHE?